cloud_service/upgrading-cloud/upgrade-cloudsvc-operator.adoc

```diff
@@ -0,0 +1,44 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="upgrade-cloudsvc-operator"]
+= Upgrading secured clusters in RHACS Cloud Service by using the Operator
+include::modules/common-attributes.adoc[]
+:context: upgrade-cloudsvc-operator
+
+toc::[]
+
+[role="_abstract"]
+
+Red{nbsp}Hat provides regular service updates for the components that it manages, including Central services. These service updates include upgrades to new versions of {product-title-managed}.
+
+You are responsible for regularly upgrading the version of {product-title-short} on your secured clusters to ensure compatibility with {product-title-managed-short}.
+
+include::modules/prepare-operator-upgrades.adoc[leveloffset=+1]
+include::modules/change-collection-method.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* link:https://docs.openshift.com/container-platform/latest/operators/admin/olm-upgrading-operators.html[Updating installed Operators]
+
+//does this apply to Cloud Service? Or only updating Central components?
```

Comment:
Good question. I doubt that there are ACS CS customers that have secured clusters on the 3.7x versions (possibly some remaining from the "service preview" in late 2022), so we could skip this part about changing channels.